src/test/java/dev/jbang/cli/TestRun.java

```diff
@@ -521,7 +521,7 @@ void testAliasWithRepo(@TempDir File output) throws IOException {
 				"  \"aliases\": {\n" +
 				"    \"aliaswithrepo\": {\n" +
 				"      \"script-ref\": \"dummygroup:dummyart:0.1\",\n" +
-				"      \"repositories\": [ \"http://dummyrepo\" ]\n" +
+				"      \"repositories\": [ \"https://dummyrepo\" ]\n" +
```

Comment:
is http now allowed anymore or ?

Reply:
what is allowed (only HTTPS or HTTP) or not is matter of setup/config. For example if you have something like this in your settings.xml (user or system): https://github.com/apache/maven/blob/master/apache-maven/src/assembly/maven/conf/settings.xml#L160-L166 then HTTP is blocked.